mentors/ruby/c.json

```diff
@@ -1 +1,10 @@
-[]
+[  
+  {
+    "github_username": "corroded",
+    "name": "Eumir Gaspar",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I love Ruby and am very passionate about it. I have used Ruby (and Rails) professionally since 2009 and have maintained the RubyConf Philippines site as a volunteer since 2014 and have"
```

Comment:
The end of the sentence seems to have been cut short :)

Reply:
true! that was weird. Hmm maybe because I edited it in Github. Anyway will change! thanks for spotting that!